make/matlab.mak

```diff
@@ -14,7 +14,8 @@
 # the empty string disables MATLAB support.
 
 MEX ?= mex
-MATLAB_PATH ?= $(subst /bin/mex,,$(realpath $(shell which '$(MEX)')))
+#MATLAB_PATH ?= $(subst /bin/mex,,$(realpath $(shell which '$(MEX)')))
```

Comment:
Really appreciate this patch; fixed the problem I was having. However, you may want to undo this change for the pull request! It points the MATLAB path to your personal location.

Reply:
Really appreciate too！